eternalcore-core/src/main/java/com/eternalcode/core/feature/jail/JailCommand.java

```diff
@@ -0,0 +1,287 @@
+package com.eternalcode.core.feature.jail;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.annotations.scan.feature.FeatureDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import java.time.Duration;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+
+@Command(name = "jail")
+@FeatureDocs(
+    name = "Jail",
+    permission = {"eternalcore.jail.bypass"},
+    description = "Permissions allow for you to setup jail location on your server, detain players, release players, and bypass jail."
```

Comment:
This feature you can rename to "Jail Bypass" because, "eternalcore.jail.bypass" permission doesn't allow to "setup, detain players, release players"